kotlin/src/main/kotlin/com/linecorp/armeria/internal/common/kotlin/ArmeriaCoroutineUtil.kt

```diff
@@ -15,14 +15,18 @@
  */
 
 @file:JvmName("ArmeriaCoroutineUtil")
+@file:Suppress("unused")
 
 package com.linecorp.armeria.internal.common.kotlin
 
 import com.linecorp.armeria.common.RequestContext
 import com.linecorp.armeria.common.kotlin.CoroutineContexts
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.future.future
+import kotlinx.coroutines.reactive.asPublisher
```

Comment:
How about directly collect `Flow` without depending on `reactive.asPublisher`?
I don't want to add an additional dependency for converting `Flow` to `Publisher`.